src/lib/edge-functions/registry.mjs

```diff
@@ -25,13 +25,15 @@ import { NETLIFYDEVERR, NETLIFYDEVLOG, chalk, log, warn, watchDebounced } from '
  * @type {object}
  * @property {string} function
  * @property {string} path
+ * @property {string=} name
  */
 
 /**
  * @typedef EdgeFunctionDeclarationWithPattern
  * @type {object}
  * @property {string} function
  * @property {string} pattern
+ * @property {string=} name
```

Comment:
TIL about optional parameters in JSDoc types!

Reply:
Saaaaaaameee